frontend/dialogs/OBSIdianPlayground.hpp

```diff
@@ -0,0 +1,34 @@
+/******************************************************************************
+    Copyright (C) 2023 by Dennis Sädtler <dennis@obsproject.com>
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+******************************************************************************/
+
+#pragma once
+
+#include <ui_OBSIdianPlayground.h>
```

Comment:
In my experience those UI headers play by their own rules - according to our rules they should be placed up here, but I've encountered quite a few situations where I _had_ to place it at the very bottom of includes.

YMMV, but just wanted to point this out that it wouldn't be "wrong" to put this at the bottom of the includes for that reason.

Reply:
I hadn't encountered any issues thus far, but moved it to the bottom just to be sure.